external-crates/move/crates/move-cli/tests/tracing_tests/tracing-unit-tests/args.exp

```diff
@@ -30,8 +30,6 @@ Running Move unit tests
 [ PASS    ] 0x1::references::test_vector_mut_borrow_pop
 Test result: OK. Total tests: 26; passed: 26; failed: 0
 External Command `diff -qr traces saved_traces`:
-Only in traces: 0x1__differential_test__f1.json.zst
```

Comment:
@tzakian is this correct? why?

Reply:
Yea, this is correct. I think this was from a bad snapshot update (new test was added and baseline updated, but `traces` wasn't copied over to `saved_traces` so `traces` had the two new files).